Oxford Reference.js

```diff
@@ -0,0 +1,226 @@
+{
+	"translatorID": "62415874-b53c-4afd-86e8-814e18a986f6",
+	"label": "Oxford Reference",
+	"creator": "Sonali Gupta",
+	"target": "http://www.oxfordreference.com/",
```

Comment:
Start with `^https?` and escape the points, which means `\.` in Scaffold or `\\.` in the textfile directly.